hips/hip-999.md

```diff
@@ -0,0 +1,105 @@
+---
+hip: 9999
+title: "Helm Org Maintainers and Deprecated Projects"
+authors: [ "Matt Butcher <matt.butcher@microsoft.com>" ]
+created: "2020-09-24"
+type: "process"
+status: "draft"
+---
+
+## Abstract
+
+This process proposal explains what happens to Helm organization maintainer slots when
+a project is deprecated.
+
+## Motivation
+
+The current governance documents state that org maintainers may be selected from any of
+the official Helm projects. However, as the Helm organization matures, some projects are
+being deprecated. The governance docs are unclear what happens to org maintainers from
+deprecated projects, and whether deprecated projects may hold org maintainer seats.
+
+## Rationale
+
+In the interest of keeping the Helm project thriving and alive, this HIP proposes that
+only non-deprecated projects are entitled to org maintainer seats.
+
+One project that is slated for deprecation is the charts repository, which previously had
+guaranteed org maintainers. This proposal updates that by splitting its designated spots
+between the remaining categories.
+
+## Specification
+
+To implement this, a new PR must be opened with the changes to the governance document.
+The changes must be adopted by 2/3 supermajority of the _current org maintainers_ 
+(including those maintainers who may be attached to deprecated projects).
+
+The governance documentation for the Helm org will be updated as follows:
+
+- All references to org maintainer seats for projects will clarify that they apply only
+  to non-deprecated projects
+- Text will clarify that when a project is deprecated, its org maintainers will
+  be moved to Emeritus status _except_ where those maintainers may be able to keep their
+  role because of their status on another Helm project, including the `helm/helm` project.
+- The distribution of voting seats will change from 3 to Helm, 2 to charts, and 2 to other
+  projects. The new distribution will be:
+  - 4 Representatives from the Helm project
+  - 3 Representatives from other Helm projects
```

Comment:
The governance document seems to describe a distribution for the _initial_ set of org maintainers: https://github.com/helm/community/blob/master/governance/governance.md#initial-org-maintainers-selection

This should probable be updated to be describe that the distribution should always be respected.

Also the governance mentions there will be between 3 and 9 org maintainers.  Should the distribution be a percentage instead?